ivy/functional/frontends/jax/lax/operators.py

```diff
@@ -647,3 +647,8 @@ def conj(x):
 @to_ivy_arrays_and_back
 def is_finite(x):
     return ivy.isfinite(x)
+
+
+@to_ivy_arrays_and_back
+def cbrt(x):
```

Comment:
Can you add the `/` argument here as it's present in the `jax.cbrt` function here :https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.cbrt.html

Reply:
Oh awesome, thanks for the explanation  :fire: